lib/alephant/logger/json.rb

```diff
@@ -41,6 +43,13 @@ def write(hash)
         @log_file.write(::JSON.generate(hash) + "\n")
       end
 
+      def writeable?(message_level)
+        Alephant::Logger::LevelsController.should_log?(
```

Comment:
I wonder if you can just reference `LevelsController.should_log?` here, as you're inside the same namespace.